web/controllers/github_api/http_client.ex

```diff
@@ -64,8 +63,12 @@ defmodule Dwylbot.GithubAPI.HTTPClient do
   def report_error(token, errors, comments_url) do
     if !Enum.empty?(errors) do
       message = errors
-      |> Enum.map(&(R.generate_message_error &1))
-      |> Enum.join("/n")
+      |> Enum.map(fn(e) -> e.error_message end)
+      |> Enum.join(
+        """
+
+        """
```

Comment:
I've tried with "\n" but this is display on the issue as text. Agree that it feels a bit weird. I intend to refactor also the way messages are post on Github. I want to have more independent rules (see #74) so I think I will post one comment per errors instead of combining multiple errors in one comment which will resolve this issue